modules/ROOT/pages/provisioning-gcp.adoc

```diff
@@ -0,0 +1,56 @@
+= Provisioning Fedora CoreOS on Google Cloud Platform
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on Google Cloud Platform (GCP).
+
+== Prerequisites
+
+Before provisioning a FCOS instance, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a GCP account. The examples below use the https://cloud.google.com/sdk/gcloud[gcloud] command-line tool, which must be separately installed and configured beforehand.
+
+== Selecting an image family
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+
+FCOS images are published under the `fedora-coreos-cloud` project and further organized into images families, tracking the tip of the corresponding stream:
+
+ * `fedora-coreos-stable`
+ * `fedora-coreos-testing`
+ * `fedora-coreos-next`
+
+Before proceeding, check the details of each xref:update-streams[update stream] and pick the one most suited for your usecase.
+
+You can inspect the current state of an image family as follows:
+
+.Inspecting an image family
+[source, bash]
+----
+export STREAM='stable'
```

Comment:
`export` isn't necessary when we're just going to be using it as a shell variable.